pkg/config/config.go

```diff
@@ -201,6 +196,36 @@ func (c *ConfigInfo) SetActiveApplication(application string, project string) er
 	return nil
 }
 
+// AddApplication add  new application to the config file
+// Newly create application is NOT going to be se as Active.
+func (c *ConfigInfo) AddApplication(application string, project string) error {
+	if c.ActiveApplications == nil {
+		c.ActiveApplications = []ApplicationInfo{}
+	}
+
+	for i, app := range c.ActiveApplications {
+		// if application exists set is as Active
+		if app.Name == application && app.Project == project {
+			c.ActiveApplications[i].Active = true
+			return fmt.Errorf("unable to set add %s application, it already exists in config file", application)
+		}
+	}
+
+	// if application doesn't exists, add it as Active
```

Comment:
should this comment be here, we are not setting the application as active

Reply:
correct, I  forgot to update that comment.

fixed